packages/shared/components/statistic-item/statistic-item.tsx

```diff
@@ -64,15 +65,20 @@ const _StatisticItem: React.FC<IFollowStatisticItemProps> = ({
         className
       )}
     >
-      {label && (
-        <div
-          className={
-            "statistics-item__top " + generateClasses(ITEM.LABEL, invert)
-          }
-        >
-          {label}
-        </div>
-      )}
+      {label ? (
+        labelTooltip && label ? (
```

Comment:
This is bad.
Try to find solution that will look good and be well supported.

Reply:
Reverted changing in StatisticItem and added new component TooltipLabel which is used with prop of label